header.php

```diff
@@ -78,12 +78,15 @@
 							/* translators: %s: name of network */
 							sprintf( __( 'Logo for %s', 'pressbooks-book' ), get_bloginfo( 'name', 'display' ) )
 						);
-						restore_current_blog();
+					} else {
 						?>
-					<?php } else { ?>
 					<svg class="header__logo--svg" role="img">
 						<use href="#logo-pressbooks" />
-					</svg><?php } ?>
+					</svg>
+						<?php
+					}
+					restore_current_blog();
```

Comment:
restore_current_blog has to be moved outside of the if/else block since switch_to_blog now happens before entering the if condition